tests/integration/API/JSON/ResponseBuilders/SegmentsResponseBuilderTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace MailPoet\API\JSON\ResponseBuilders;
+
+use MailPoet\DI\ContainerWrapper;
+use MailPoet\Entities\SegmentEntity;
+use MailPoetVendor\Doctrine\ORM\EntityManager;
+
+class SegmentsResponseBuilderTest extends \MailPoetTest {
+  public function testItBuildsResponse() {
+    $name = 'Response Builder Test';
+    $description = 'Testing description';
+
+    $di = ContainerWrapper::getInstance();
+    $em = $di->get(EntityManager::class);
+    $em->persist($segment = new SegmentEntity($name, SegmentEntity::TYPE_DEFAULT, $description));
```

Comment:
I would abstain from assignment in function arguments as it makes a line less readable.

Reply:
Good point, I agree with you.
I was inspired by another test where is used a similar code style https://github.com/mailpoet/mailpoet/blob/master/tests/integration/API/JSON/ResponseBuilders/NewslettersResponseBuilderTest.php#L16

I'll fix it.